.github/workflows/build-test

```diff
@@ -104,7 +104,12 @@ for MODULE in $MODULES_TO_TEST; do
                 dotnet iqsharp install --user
             fi
 
-            pytest --doctest-modules
+            # Disable the remote tests for pytket-aqt, -ionq, and -qsharp for now
+            if [[ "${MODULE}" = "pytket-aqt" || "${MODULE}" = "pytket-ionq" || "${MODULE}" = "pytket-qsharp" ]]; then
```

Comment:
The pytket-qsharp remote tests are now working for me. Can we un-disable them here?